docker-compose.yml

```diff
@@ -1,254 +1,253 @@
+
+  #
+  # mender-workflows-server
+  #
+  mender-workflows-server:
+    image: ${MENDER_REGISTRY}/${MENDER_REPOSITORY}/workflows:${MENDER_TAG}
+    environment:
+      WORKFLOWS_MONGO_URL: mongodb://mender-mongo:27017
+    extends:
+      file: common.yml
+      service: mender-base
+    networks:
+      - mender
+    depends_on:
+      - mender-mongo
+      - mender-nats
+    labels:
+      mender.healthcheck.path: "/health"
+
+  #
+  # mender-workflows-worker
+  #
+  mender-workflows-worker:
+    image: ${MENDER_REGISTRY}/${MENDER_REPOSITORY}/workflows${MENDER_LEGACY_IMAGE:+-worker}:${MENDER_TAG}
+    command:
+      - worker
+      - --excluded-workflows=generate_artifact,generate_delta_artifact
+      - --automigrate
+    environment:
+      WORKFLOWS_MONGO_URL: mongodb://mender-mongo:27017
+      DEVICECONNECT_ADDR: "mender-deviceconnect:8080"
+      HAVE_DEVICECONNECT: 1
+      HAVE_DEVICECONFIG: 1
+      DEVICECONFIG_ADDR: "mender-deviceconfig:8080"
+    extends:
+      file: common.yml
+      service: mender-base
+    networks:
+      - mender
+    depends_on:
+      - mender-mongo
+      - mender-nats
+
+  #
+  # mender-create-artifact-worker
+  #
+  mender-create-artifact-worker:
+    image: ${MENDER_REGISTRY}/${MENDER_REPOSITORY}/create-artifact-worker:${MENDER_TAG}
+    extends:
+      file: common.yml
+      service: mender-base
+    environment:
+      - CREATE_ARTIFACT_DEPLOYMENTS_URL=http://mender-deployments:8080
+    networks:
+      - mender
+    depends_on:
+      - mender-mongo
+      - mender-nats
+
+  #
+  # mender-deviceconnect
+  #
+  mender-deviceconnect:
+    image: ${MENDER_REGISTRY}/${MENDER_REPOSITORY}/deviceconnect:${MENDER_TAG}
+    command: server --automigrate
+    extends:
+      file: common.yml
+      service: mender-base
+    networks:
+      - mender
+    depends_on:
+      - mender-mongo
+      - mender-nats
+    environment:
+      DEVICECONNECT_MONGO_URL: "mongodb://mender-mongo"
+      DEVICECONNECT_NATS_URI: "nats://mender-nats:4222"
+    labels:
+      mender.healthcheck.path: "/api/internal/v1/deviceconnect/health"
+
+  #
+  # mender-deviceconfig
+  #
+  mender-deviceconfig:
+    image: ${MENDER_REGISTRY}/${MENDER_REPOSITORY}/deviceconfig:${MENDER_TAG}
+    extends:
+      file: common.yml
+      service: mender-base
+    networks:
+      - mender
+    depends_on:
+      - mender-mongo
+    command: server --automigrate
+    labels:
+      mender.healthcheck.path: "/api/internal/v1/deviceconfig/health"
+
+  mender-mongo:
+    image: mongo:6.0
+    extends:
+      file: common.yml
+      service: mender-base
+    networks:
+      mender:
+        aliases:
+          - mongo-tenantadm
+          - mongo-deployments
+          - mongo-device-auth
+          - mongo-inventory
+          - mongo-useradm
+          - mongo-workflows
+
+  mender-nats:
+    image: nats:2.7.4-scratch
+    command: -js
+    networks:
+      - mender
 
 networks:
-    mender: null
+  mender: {}
```

Comment:
So what is this commit doing besides formatting? I don't get the "Complete the client compose files"

Reply:
The change is primarily to the other compose files. Making it possible to evaluate them without having to pull in the main `docker-compose.yml` file. I used it in the following commit when evaluating images when doing the `extract_fs` dance :dancers: